micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jpa/HibernateMetrics.java

```diff
@@ -47,6 +54,9 @@
     @Nullable
     private final Statistics statistics;
 
+    @Nullable
```

Comment:
Is this Nullable? If the `sessionFactory` passed in the constructor is `null` then a NPE will happen: `Statistics statistics = sessionFactory.getStatistics();`

Reply:
You're right. WIll change to @NonNull

Reply:
//EDIT
This field is Nullable when using the other Deprecated Constructor. NPE in this constructor won't happen because of NonNullApi annotation.